tests/manage/z_cluster/cluster_exp_helpers.py

```diff
@@ -0,0 +1,215 @@
+import logging
+from concurrent.futures import ThreadPoolExecutor
+from uuid import uuid4
+
+from ocs_ci.ocs import constants
+import ocs_ci.ocs.resources.pod as pod_helpers
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.cluster import get_percent_used_capacity
+from ocs_ci.ocs import node
+
+logger = logging.getLogger(__name__)
+
+
+def raw_block_io(raw_blk_pod, size='10G'):
+    """
+    Runs the block ios on pod baased raw block pvc
+    Args:
+        raw_blk_pod(pod): pod on which  block IOs should run
+        size(str): IO size
+
+    Returns:
+        bool : True if IOs happen successfully
+
+    """
+    raw_blk_pod.run_io(storage_type='block', size=size)
+    return True
+
+
+def downloader(dest_pod, namespace):
```

Comment:
why is this function needed? Can't we just call pod.upload() directly?

Reply:
ack